src/index/writer.rs

```diff
@@ -10,16 +10,22 @@
 
 use crate::byte::ByteWriter;
 use crate::client_factory::ClientFactoryAsync;
-use crate::index::{Fields, IndexRecord, RECORD_SIZE};
+use crate::index::{Fields, IndexRecord};
 
 use pravega_client_shared::ScopedStream;
 
+use crate::segment::raw_client::RawClient;
+use crate::util::get_request_id;
 use bincode2::Error as BincodeError;
+use pravega_wire_protocol::commands::UpdateSegmentAttributeCommand;
+use pravega_wire_protocol::wire_commands::Requests;
 use snafu::{ensure, Backtrace, ResultExt, Snafu};
 use std::fmt::Debug;
 use std::marker::PhantomData;
+use tracing::info;
 
 const MAX_FIELDS_SIZE: usize = 100;
+pub const INDEX_RECORD_SIZE_ATTRIBUTE_ID: u128 = 80000000000000000000000000000011;
```

Comment:
Is this number supposed to be in hex? Because it is not. (It should start with `0x`) Also use some `_` characters to make it more readable. Also add a comment for where it comes from.